scripts/lib.sh

```diff
@@ -35,7 +35,9 @@ function assume_role() {
 
 	# Unset AWS credential environment variables so they don't interfere with `assume-role`
 	unset AWS_ACCESS_KEY_ID
-	unset AWS_SECRET_ACCES_KEY
+	unset AWS_SECRET_ACCESS_KEY
+	## fix for https://github.com/cloudposse/reference-architectures/issues/8
+	export AWS_SHARED_CREDENTIALS_FILE="${HOME}/.aws/credentials"
```

Comment:
Please move to line ~35 below `AWS_CONFIG_FILE` since it's logically relatd to that.